pkg/inject/patch_test.go

```diff
@@ -34,16 +34,14 @@ func TestPatch(t *testing.T) {
 		createdBy           = "linkerd/cli v18.8.4"
 	)
 
-	actual := NewPatchDeployment()
+	actual := NewPatch(k8sPkg.Deployment)
 	actual.addContainer(sidecar)
 	actual.addInitContainerRoot()
 	actual.addInitContainer(init)
-	//actual.addVolumeRoot()
-	//actual.addVolume(secrets)
```

Comment:
I'd actually recommend uncommenting these lines, instead of removing them. And also uncommenting lines 16-21 and 49-50 of this file, so that we can start verifying that the volume patches are working correctly. You'd need to create a new secret, however, in order for this to work. See https://github.com/linkerd/linkerd2/commit/91c5f07650c3dc25e6cab0134e6a3185e8ec20b4#diff-4940a4b4cb3bf0d6f5d4bf9090e2ea6e, which shows when the original tests were removed.